tests/js/unit/utility/colorUtilitySpec.js

```diff
@@ -96,10 +96,10 @@ describe('ColorUtility shared tests', function () {
 
 	it('should call forward the call to hslToRgb', function () {
 		expect(ColorUtility._hslToRgb(6, 70, 68));
-		expect(hslToRgb).toHaveBeenCalledWith(6, 70, 68);
+		expect(hslToRgb).toHaveBeenCalledWith(6, .7, .68);
```

Comment:
JSHint shows warnings and the grunt tasks fails, so you should add a leading 0 I guess.
> A leading decimal point can be confused with a dot